src/sidebar/components/StreamSearchInput.tsx

```diff
@@ -14,15 +14,15 @@ export type StreamSearchInputProps = {
  */
 function StreamSearchInput({ router }: StreamSearchInputProps) {
   const store = useSidebarStore();
-  const query = store.routeParams().q;
+  const { q } = store.routeParams();
```

Comment:
```suggestion
  const { q: query } = store.routeParams();
```

Reply:
Actually, this was partially intentional, because the `setQuery` function below has a `query` argument, which was shadowing the `query` param from outer scope.

At first it made me think it was the same param and tried to add the null coalescing there as well, so I thought using different names would make it more obvious for the next person landing there.

Also, less important, but using `query ?? ''` in the component below instead of `q ?? ''` makes the line longer than 80 characters and prettier collapses it in multiple lines 😅